server/paymentProviders/opencollective/host.js

```diff
@@ -16,6 +16,20 @@ paymentMethodProvider.getBalance = () => {
   return Promise.resolve(maxInteger);
 };
 
+paymentMethodProvider.createPlatformTipTransaction = async transaction => {
```

Comment:
We already have a function for that called `Transaction.createFeesOnTopTransaction`.
That function is automatically called by `Transaction.createFromPayload` if `transaction.data?.isFeesOnTop && transaction.platformFeeInHostCurrency`.

Do we really need this logic here?

Reply:
@kewitz : My understanding is that the `createFeesOnTopTransaction` is done when we have a transaction and some fee on top of it. So we send the amount and the fee in a single transaction. However as per the new changes to the front-end what we have now is two sections of the modal. Where the first request sends the transaction amount which goes from the source to the collective. And the platform tip is sent separately in the second part of the modal. In this case the `amount` is undefined and the platform tip should go from the host to opencollective which I think feesOnTop doesn't handle). Which is why I am having this function. Does that make sense? Maybe we'll arrange a call to clarify this further. 😄